### ISSUE TITLE
Remote Wiring Diagram

### ISSUE THREAD

**Issue by twodawg** [XRobots/openDogV2#1]
![15964148463708307811105749923681](https://user-images.githubusercontent.com/1840110/89136032-675df980-d4e6-11ea-9f58-9fb807d711f7.jpg)

I used the YouTube video with the open remote case and the remote source code to try and plan the wiring.  The code for the buttons and switches does not seem to match the YouTube video.  Should I modify the code or switch the wiring to match the code? 

Did I make any mistakes with the wiring?

Here is my build with slightly different parts.
![1596414428123590616221710997594](https://user-images.githubusercontent.com/1840110/89135900-90ca5580-d4e5-11ea-8449-76a3ca0f0468.jpg)

I apologize for the image rotation. My attempts to rotate it have failed.

**Comment by XRobots** [comment]
Yes not all the switches are used, and some variables are named weirdly.

Essentially, the two switches on the right hand side are toggleTop and toggleBottom, but they get read as but4 and but5. The three buttons are menuUp/Down/Select/ . And then I only used two of the three upper switches which end up as  toggle1 and toggle2, but they get read as sw14 and sw2 before the if statements turn them into their variables in the data struct.
